R/utils.R

```diff
@@ -16,6 +16,14 @@ lockbox_imports <- function() {
   Filter(nzchar, strsplit(read.dcf(dcf)[,"Imports"], "[\n, ]+")[[1]])
 }
 
+announce_package_usage <- function(name, version, dependency = FALSE) {
```

Comment:
unless you want to pass verbose argument here set to FALSE by default, and then call this function with verbose = TRUE explicitly if you want things printed